src/legacy/core_plugins/expressions/public/np_ready/public/loader.ts

```diff
@@ -68,10 +68,18 @@ export class ExpressionLoader {
       extraHandlers: params.extraHandlers,
     };
 
-    this.execute(expression, params);
+    this.loadingSubject.next();
```

Comment:
is there a reason this is added back ? (it was removed on your request in my initial pr)

Reply:
I was trying to get the loading spinner to show in all cases, and it looks like on the initial render we were not showing it.

Reply:
on initial render loading status should be true .... so you don't need to wait for this event.